API-Reference/Kokkos::create_mirror.md

```diff
@@ -7,31 +7,44 @@ A common desired use case is to have a memory allocation in GPU memory and an id
 Usage:
 ```cpp
 auto host_mirror = create_mirror(a_view);
-auto host_mirror_view = create_mirror_view(a_view); 
+auto host_mirror_view = create_mirror_view(a_view);
 
 auto host_mirror_space = create_mirror(ExecSpace(),a_view);
-auto host_mirror_view_space = create_mirror_view(ExecSpace(),a_view); 
+auto host_mirror_view_space = create_mirror_view(ExecSpace(),a_view);
 ```
 
-## Synopsis 
+## Synopsis
 
 ```cpp
 
 template <class ViewType>
 typename ViewType::HostMirror create_mirror(ViewType const&);
```

Comment:
I just opened a new issue to note that we should document synchronization semantics of all these
https://github.com/kokkos/kokkos/issues/4837